sdk/python/tests/integration/e2e/test_universal_e2e.py

```diff
@@ -8,34 +8,42 @@
 
 from feast import FeatureStore, FeatureView
 from tests.integration.feature_repos.universal.entities import driver
-from tests.integration.feature_repos.universal.feature_views import driver_feature_view
+from tests.integration.feature_repos.universal.feature_views import (
+    driver_feature_view,
+    global_feature_view,
+)
 
 
 @pytest.mark.integration
 @pytest.mark.parametrize("infer_features", [True, False])
 def test_e2e_consistency(environment, e2e_data_sources, infer_features):
     fs = environment.feature_store
-    df, data_source = e2e_data_sources
+    df, data_source, entityless_data_source = e2e_data_sources
```

Comment:
There are a few confusing things here
* Where is `df` used? I know you didn't introduce it, but it's probably worth figuring out what the intention is behind `e2e_data_sources`.
* It's not clear to me how `e2e_data_sources` is supposed to be different from the universal data sources. Is there anything preventing us from adding the entityless df/view/source as a part of the universal repo?

Reply:
`df` isn't used in this case.

`e2e_data_sources` is a data source that is created with a handful of entries with specific timestamps, specifically for the purpose of validating the point in time correctness of the feature retrievals in the online and offline case. 

The universal data sources are generated randomly, and with an arbitrary number of records that can be specified by users. but there's no logic for time generating records with specific kinds of relative timestamps.

We should consolidate the two datasets for our internal tests, but I do imagine that we may still have the need for multiple kinds of datasets to check the correctness of specific features.

For this specific PR, do we need to add any of the time correctness checks for entityless feature views? If not, I'd leave this file untouched.

Reply:
Yeah that sounds good to me. I think `test_universal_historical` and `test_universal_online` will have enough coverage.